### PR TITLE
Added validator btc addresses on format P2PKH

### PR DESCRIPTION
**Pull request by daveusa31**
Hello. Please tell me what I did wrong?
Why doesn't travic-ci check?

### CODE REVIEW

**Comment by kvesteri** [comment]
@daveusa31 There are flake8 and isort errors. You can see the errors in the travis report.

**Comment by daveusa31** [comment]
I'm sorry, but I couldn't figure it out :(

**Comment by kvesteri** [comment]
You can check the errors locally by first installing flake8 and isort
```
>> pip install isort
>> pip install flake8 
```

The you can run the following command for isort:

```
>> isort --recursive --diff validators tests && isort --recursive --check-only validators tests
```
And for flake8

```
flake8 validators tests
```

**Comment by daveusa31** [comment]
Yeah, it works

File: validators/btc_address.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import re
+
+from .utils import validator
+
+pattern = re.compile(r"^[13][a-km-zA-HJ-NP-Z1-9]{25,34}$")
+
+
+@validator
+def btc_address(value):
```
Comment (kvesteri) [inline validators/btc_address.py:9]: Please add a doc block here defining how this function is to be used. See other functions of similar doc blocks. Also reference this doc block in the ``index.rst`` file

**Comment by daveusa31** [comment]
Added

**Pull request merged**